pkg/flare/cluster_checks.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 )
 
+const kubeEndpointsProviderName string = "kube_endpoints"
```

Comment:
what I was thinking is to use the same const string every where it the code source:
- https://github.com/DataDog/datadog-agent/blob/dfa9f99d4da39f40bd52ce919a76ee80b671ddc9/pkg/autodiscovery/providers/kube_endpoints.go#L211